geomesa-tools/README.md

```diff
@@ -412,6 +412,8 @@ To export features, use the `export` command.
         -F, --format
            Format to export (csv|tsv|gml|json|shp|bin)
            Default: csv
+        --gzip
+           level of gzip compression to apply to output, from 1-9
```

Comment:
batching is good like you mentioned in chat...later is OK...I imagine the ability to specify 64 or 128 MB blocks for HDFS would be useful